ci-operator/step-registry/rhba/interop-tests/rhba-interop-tests-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export TEST_COLLECT_BASE_DIR=${ARTIFACT_DIR}
+
+echo "Running tests..."
+/opt/runTest.sh
```

Comment:
Does runTest.sh exit with non-zero if tests fail?  If so the script will exit and the following code will not be executed and results will not have the junit_ prefix

Reply:
yes this will exit with non-zero if tests fail. The test execution is done in same way as it was in previous interop pipeline. Tests are run using maven and failsafe-reports are copied to results as well.